packages/next/src/telemetry/.vercel.approvers

```diff
@@ -0,0 +1 @@
+@padmaia
```

Comment:
I would recommend including at least 3 people so that PRs to this directory aren't blocked if this person is OOO or unavailable.

Reply:
Good point, I've updated it to match the old config.